server/src/main/java/org/opensearch/action/admin/cluster/node/stats/NodesStatsRequest.java

```diff
@@ -80,6 +86,7 @@ public NodesStatsRequest(String... nodesIds) {
     public NodesStatsRequest all() {
         this.indices.all();
         this.requestedMetrics.addAll(Metric.allMetrics());
+        this.clearRestActionsFilters();
```

Comment:
why are we calling clearRestActionsFilters() here?
Also, should we have a check for Version.V_3_0_0 before we update the RestActionsFilters?

Reply:
We are calling `clearRestActionsFilters()` here to clear out any filters if at all present so that `all` will return the stats for all the action instead of filtering out

> Also, should we have a check for Version.V_3_0_0 before we update the RestActionsFilters?

Since we're not doing any inter node operation of updating rest action, backward compatibility won't be any issue and hence check is not required